### PR TITLE
core: update scripts with automatic app discovery loops

### PR DESCRIPTION
**Pull request by Kaian**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

File: library/bin/test-psalm [hunk 1]
```diff
@@ -2,52 +2,11 @@
...
-pushd /opt/irontec/ivozprovider/microservices/scheduler
-    echo "=========== microservices/scheduler ==========="
-    ./vendor/bin/psalm --threads=2 --config=psalm.xml $*
-popd
-
-pushd /opt/irontec/ivozprovider/microservices/workers
-    echo "=========== microservices/workers ==========="
-    ./vendor/bin/psalm --threads=2 --config=psalm.xml $*
-popd
+APPS=$(find /opt/irontec/ivozprovider -name composer.json ! -path '*/vendor/*' ! -path '*/composer-packages/*' ! -path "*/agi/*" ! -path "*/schema/*" ! -path "*/realtime/*" -printf "%h\n")
```
Comment (Kaian) [inline library/bin/test-psalm:5]: psalm is not executed in following apps:

- schema
- microservices/realtime
- asterisk/agi

File: library/bin/test-phpstan [hunk 2]
```diff
@@ -5,33 +5,21 @@ set -e
 HOME=/opt/irontec/ivozprovider
 cd $HOME
 
-run_test () {
-    local APP_ROOT="$HOME/$1"
-    local TARGET=$2
+APPS=$(find $HOME -name composer.json ! -path '*/vendor/*' ! -path '*/composer-packages/*' ! -path "*/schema/*" -printf "%h\n")
```
Comment (Kaian) [inline library/bin/test-phpstan:8]: phpstan is not executed in schema app

File: library/bin/composer-require [hunk 3]
```diff
@@ -1,53 +1,14 @@
...
-popd
-
-pushd /opt/irontec/ivozprovider/microservices/provision
-    composer require $*
-popd
-
-pushd /opt/irontec/ivozprovider/microservices/scheduler
-    composer require $*
-popd
+APPS=$(find /opt/irontec/ivozprovider -name composer.json ! -path '*/vendor/*' ! -path '*/composer-packages/*' -printf "%h\n")
```
Comment (mmadariaga) [inline library/bin/composer-require:8]: exclude library

File: library/bin/test-phpstan (hunk 2 above)
Comment (mmadariaga) [reply library/bin/test-phpstan:8]: 🤔

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: library/bin/composer-require (hunk 3 above)
Comment (Kaian) [reply library/bin/composer-require:8]: Good catch!

Comment (Kaian) [reply library/bin/composer-require]: Updated

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**